src/pyuvdata/telescopes.py

```diff
@@ -17,12 +17,11 @@
 from astropy import units
 from astropy.coordinates import Angle, EarthLocation
 
+from pyuvdata import hdf5_utils
+from pyuvdata import parameter as uvp
+from pyuvdata import utils as uvutils
```

Comment:
it's not clear to me why these had to be changed? Relative imports still work in src layout

Reply:
I can put them back. I was changing them in tests and I guess got overzealous. I don't personally have a strong preference for either way, so happy to go with relative.

Reply:
There were cases in some files that had lots of leading dots which I find challenging to read, and even in this file there was a mix of relative and absolute imports (`from pyuvdata.data import DATA_PATH`). I think I made these changes because the mix of absolute and relative felt weird so I made them all look similar. Do you have thoughts about the mixture?

Reply:
Done.